its/src/test/java/com/sonar/it/csharp/NoSourcesTest.java

```diff
@@ -0,0 +1,50 @@
+package com.sonar.it.csharp;
+
+import com.sonar.it.shared.TestUtils;
+import com.sonar.orchestrator.Orchestrator;
+import com.sonar.orchestrator.build.ScannerForMSBuild;
+import java.nio.file.Path;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import static com.sonar.it.csharp.Tests.ORCHESTRATOR;
+import static com.sonar.it.csharp.Tests.getMeasureAsInt;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class NoSourcesTest {
+
+  @ClassRule
+  public static TemporaryFolder temp = TestUtils.createTempFolder();
+
+  private static final String PROJECT = "ProjectWithNoSources";
```

Comment:
I'd prefer to have those two `@ClassRule` together and this constant above/below them. Current order and empty lines seems irregular.